### PR TITLE
Excel dump fixes and PHPExcel/README requirements

### PR DESCRIPTION
**Pull request by Jkat**
Excel dump will no longer work for 'large' datasets without caching enabled.

SQLite must be installed separately. 

Also making the excel dump more generic by removing hardcoded sites from projects.

### CODE REVIEW

**Comment by Jkat** [comment]
Adding the 'Discussion Required' label because I don't know what the CentOS equivalent is and to my knowledge, this caching feature has not been tested on CentOS.

Also, not sure if SQLite must be installed separately for PHP 5.6 / 7 in order for this to work.

Also officially, PHP 5.3.X is the latest supported version on CentOS 6, and PHP 5.4.X for CentOS 7. I'll leave it up to you(yes, you) on how you want to interpret that.

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ LORIS is a web-accessible database solution for neuroimaging, providing a secure
  * LINUX (supported on Ubuntu 14.04 and CentOS 6.5) or Mac OS X (tested for Mavericks - OS X 10.9)
  * Apache2 (libapache2-mod-php5)
  * MySQL (libmysqlclient15-dev mysql-client mysql-server)
- * PHP/PEAR 5.3+ (php5 php-pear php5-mysql php5-gd)
+ * PHP 5.3+ (php5 php5-mysql php5-gd php5-sqlite)
+ * PEAR (php-pear)
```
Comment (tedstrauss) [inline README.md]: I thought we were deprecating PEAR, no?

File: tools/excelDump.php [hunk 2]
```diff
@@ -101,25 +101,11 @@ function MapSubprojectID(&$results) {
...
-	writeExcel($Test_name, $instrument_table, $dataDir);
-}
-
-/*
 * Candidate Information query
 */
 $Test_name = "candidate_info";
 //this query is a but clunky, but it gets rid of all the crap that would otherwise appear.
-$query = "select distinct c.PSCID, c.CandID, c.Gender, c.DoB, s.SubprojectID from candidate c, session s where c.CandID = s.CandID and substring(c.PSCID, 1, 3) in ('PHI', 'STL', 'SEA', 'UNC') and c.Active='Y' order by c.PSCID";
+$query = "select distinct c.PSCID, c.CandID, c.Gender, c.DoB, s.SubprojectID from candidate c, session s where c.CandID = s.CandID and c.Active='Y' order by c.PSCID";
```
Comment (tedstrauss) [inline tools/excelDump.php]: wouldn't this be more useful for the user?

`select distinct c.PSCID, c.CandID, c.Gender, c.DoB, p.title from candidate c, session s, subproject p where c.CandID = s.CandID and s.SubprojectID = p.SubprojectID and c.Active='Y' order by c.PSCID`

File: README.md (hunk 1 above)
Comment (Jkat) [inline README.md]: Are we? I still see 'pear' listed under 'repositories' in our composer.json
It's also throughout our codebase like our install script, dockerfile, etc so if we are, we can send a separate pull request to get rid of all pear stuff. I don't want to mix that too much into this pull request.

File: tools/excelDump.php (hunk 2 above)
Comment (Jkat) [inline tools/excelDump.php]: I didn't look too much into this query, all I was trying to do was get rid of project specific stuff. If someone has investigated to come up with a better query, please send a pull request either based on mine or a separate one.

File: README.md (hunk 1 above)
Comment (driusan) [inline README.md]: I don't think it should be there in the composer.json (I think it was required for the old SpreadsheetWriter?), but I agree that it should be a different pull request and not mixed into this unrelated one (also our official policy is going to be PHP 5.6+ for this release, so that needs to be updated separately..)

File: tools/excelDump.php (hunk 2 above)
Comment (driusan) [inline tools/excelDump.php]: I think the way to keep the same logic without being project specific is to add "and s.CenterID <> 1" (to exclude DCC) and add a check for psc.Study_site='Y'

Comment (Jkat) [inline tools/excelDump.php]: @driusan, I did a lazy version based on what you said. 
Partly because my memory tells me that one day we're changing psc.Study_site and possibly some other similar fields...
